services/repository/repository.go

```diff
@@ -64,12 +64,13 @@ func DeleteRepository(doer *models.User, repo *models.Repository) error {
 		log.Error("CloseRepoBranchesPulls failed: %v", err)
 	}
 
+	// If the repo has a webhook, we need to trigger it before deleting it...
+	notification.NotifyDeleteRepository(doer, repo)
+
 	if err := models.DeleteRepository(doer, repo.OwnerID, repo.ID); err != nil {
```

Comment:
Thanks for PR :) Seems our linter is saying that this if conditional can be changed to just `return models.DeleteRepository(doer, repo.OwnerID, repo.ID)` and the `return nil` below could also be removed.

Reply:
Cheers, just seen! `make lint-backend` unfortunately doesn't work for me on Windows

Reply:
Awesome, thanks :) the linter has been appeased. Sorry about the developer experience on windows.

Reply:
Aah no worries, I'll have a little poke around. I don't know whether it's Windows or Git Bash, but there's some funk going on with the slashes:
![image](https://user-images.githubusercontent.com/7294642/94909008-6facc380-049a-11eb-932e-6f57f535888a.png)